TESTING.md

```diff
@@ -99,24 +99,17 @@ Marked with a github issue label "manual testing."
 
 **2.10** Click **OK** to close **Project Structure** window.
 
-### 3 Observe startup notification
-
-**3.1** ASSERTION: The screen shows a notification telling
-        you are using a pre-release version of the plugin.
-
-**3.2** ASSERTION: The version mentioned in the notification is correct.
-
-![Version notification](images/version-notification.png)
-
-- You can check the version of the plugin in **File > Settings... > Plugins > Installed**
-
```

Comment:
Nit: Chapter numbers go now 1, 2, 4, 5, ... but I guess that's okay because otherwise everything should be renumbered.

I'm wondering if we should change the numbering scheme to get rid of this problem.

Reply:
Perhaps get rid of numbers completely and just use bullet points. The beginning of the guide can mention that the steps are to be done in order if that's potentially unclear.

Reply:
`ctrl + f` & `ctrl + r` are quick an easy way to renumber (had this on todo)